include/eigensolve_quda.h

```diff
@@ -5,6 +5,9 @@
 #include <dirac_quda.h>
 #include <color_spinor_field.h>
 
+// Local enum for the LU axpy block type
+enum blockType { PENCIL, LOWER_TRI, UPPER_TRI };
```

Comment:
This should go within the `quda` namespace

Reply:
I'll fix in my next push